src/io/stun.rs

```diff
@@ -503,21 +630,28 @@ use crate::util::NonCryptographicRng;
 
 impl<'a> Attributes<'a> {
     const ALTERNATE_SERVER: u16 = 0x8023;
-    const ERROR_CODE: u16 = 0x0009;
     const FINGERPRINT: u16 = 0x8028;
     const ICE_CONTROLLED: u16 = 0x8029;
     const ICE_CONTROLLING: u16 = 0x802a;
+
     const MAPPED_ADDRESS: u16 = 0x0001;
+    const USERNAME: u16 = 0x0006;
     const MESSAGE_INTEGRITY: u16 = 0x0008;
-    const NETWORK_COST: u16 = 0xc057;
-    const NONCE: u16 = 0x0015;
-    const PRIORITY: u16 = 0x0024;
+    const ERROR_CODE: u16 = 0x0009;
```

Comment:
why does this one move?

Reply:
I re-ordered these to be in the order of their identifier

Reply:
That's all in 3fcdd83c4a28d2b86f0b7dd4c162b1a425189d04